src/parser.ts

```diff
@@ -1,481 +1,508 @@
-import * as vscode from 'vscode';
+import * as vscode from "vscode";
```

Comment:
Should not change space formatting, I look forward for this PR to be merged ! Thanks